src/index.ts

```diff
@@ -2,3 +2,4 @@ export * from './types'
 export { ResolveAddress } from './resolve-address'
 export { ResolveGroup } from './resolve-group'
 export { ResolveNom } from './resolve-nom'
+export { ResolveCelo } from './resolve-celo'
```

Comment:
will this be updated to ResolveSocialConnect?

Reply:
Yes, in a follow-on PR!